pingcap/tidb
# planner: add monitor for non-prepared plan cache

### What problem does this PR solve?


Issue Number: ref #36598 

Problem Summary:

### What is changed and how it works?
Add monitor for non-prepared plan cache in Grafana. 
Location: TiDB Cluster --> Executor --> `Queries Using Plan Cache OPS`, `Plan Cache Miss OPS`, `Plan Cache Memory Usage`, `Plan Cache Plan Num`
We use the same panels as prepared plan cache, they can be distinguished by different legend
### Check List

Tests 


Side effects


Documentation


### Release note



Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### Manual test
Start a TiDB cluster with 2 db node and set the non-prepared plan cache switch on. Cache prepared and non-prepared plans:
* TiDB-1 prepare: 1   non-prepare: 3
* TiDB-2 prepare: 2  non-prepare: 4
check the  4 monitor if works when caching plan and close the session.
<img width="637" alt="截屏2023-02-28 上午11 38 09" src="https://user-images.githubusercontent.com/108643977/221747533-e4502820-67bf-498f-a63b-7956a141cf3c.png">

<img width="638" alt="截屏2023-02-28 上午11 38 17" src="https://user-images.githubusercontent.com/108643977/221747545-9f1ae665-fb84-43f1-98d6-c45c14799073.png">

<img width="641" alt="截屏2023-02-28 上午11 40 08" src="https://user-images.githubusercontent.com/108643977/221747703-db0d32b3-6f05-4ca0-a5f1-2c1da1c0bd00.png">

<img width="641" alt="截屏2023-02-28 上午11 38 41" src="https://user-images.githubusercontent.com/108643977/221747580-30199811-e6e4-41cc-9326-45e900a44ea4.png">
